<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Algorithm</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="algorithmComboBox">
      <item>
       <property name="text">
        <string>FCFS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Round Robin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SJF Preemptive</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SJF Non-Preemptive</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Priority Preemptive</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Priority Non-Preemptive</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Process ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="idLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Arrival Time</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="arrivalLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Burst Time</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="burstLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="priorityLabel">
      <property name="text">
       <string>Priority</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PriorityLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="quantumLabel">
      <property name="text">
       <string>Quantum</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="quantumLineEdit"/>
    </item>
    <item>
     <widget class="QTableWidget" name="processTable">
      <property name="columnCount">
       <number>3</number>
      </property>
      <column>
       <property name="text">
        <string>Process id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>arrival time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>burst time </string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addProcessButton">
      <property name="text">
       <string>Add Process</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteProcessButton">
      <property name="text">
       <string>Delete Process</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_start">
      <property name="text">
       <string>start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
